launcherd/launcherd.go

```diff
@@ -1 +1,50 @@
 package launcherd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+// Launcherd a launcher daemon for starting webservices
+type Launcherd struct {
+	List []string
+}
+
+// Start starts a launcherd
+func (d *Launcherd) Start() {
+	// [ ] prepare lib (rootfs)
+	// [ ] fork and chroot each service
+	for _, service := range d.List {
+		err := d.launch(service)
+		if err != nil {
+			log.Printf("Failed to start %s", service)
+		} else {
+			log.Fatalf("Succesfully start %s", service)
+		}
+	}
+	// NEAR FUTURE
+	// [ ] set namespace and cgroups
+	// [ ] monitor health status of running service
+}
+
+func (d *Launcherd) launch(serviceName string) error {
+	// StdOut Stdin StdErr ???
+	chrPath := fmt.Sprintf("./%schroot", serviceName)
```

Comment:
suggested naming change 
```suggestion
	chrootDir := fmt.Sprintf("./%schroot", serviceName)
```
Thoughts?